src/Agent.Worker/Handlers/StepHost.cs

```diff
@@ -147,7 +151,16 @@ public string ResolvePathForStepHost(string path)
             //    We use this intermediate script to read everything from STDIN, then launch the task execution engine (node/powershell) and redirect STDOUT/STDERR
 
             string tempDir = Path.Combine(HostContext.GetDirectory(WellKnownDirectory.Work), Constants.Path.TempDirectory);
-            File.Copy(Path.Combine(HostContext.GetDirectory(WellKnownDirectory.Bin), "containerHandlerInvoker.js.template"), Path.Combine(tempDir, "containerHandlerInvoker.js"), true);
+            string targetEntryScript = Path.Combine(tempDir, "containerHandlerInvoker.js");
```

Comment:
What does switching to just the `js` version get us here? To be clear, I don't know why it would have been better with the template postfix, but just trying to understand what's going on.

Reply:
Mainly, we were not using it as a template, since we were just copying it and by making it .js, I get syntax highlighting in my editor :)

Reply:
If we ever make it a template, I would suggest "containerHandlerInvoker.template.js"